src/plugins/irc-events/whois.js

```diff
@@ -20,10 +16,20 @@ module.exports = function(irc, network) {
 				chan: chan
 			});
 		}
-		var msg = new Msg({
-			type: Msg.Type.WHOIS,
-			whois: data
-		});
+
+		var msg;
+		if (data.error) {
+			msg = new Msg({
+				type: Msg.Type.ERROR,
+				text: "No such nick: " + data.nick
```

Comment:
Is that the only type of error that can happen here, or is it worth displaying `data.error` as we do with `reason`s in some other places?